### DISCUSSION TITLE
Add "FRAMA - Supertrend strategy"

### DISCUSSION THREAD

**Discussion by alinred** [TreborNamor/TradingView-Machine-Learning-GUI#11] [General]
Are you able to add "FRAMA - Supertrend strategy" 
https://www.tradingview.com/script/y81orwSC-FRAMA-Supertrend-strategy/

Is that possible to change all values not only Take Profit and Stop Loss?

**Comment by Daat1313** [comment]
Hi! If it is possible to optimize a number of parametrs predifined in strategie and also time frames - this would be awesome.
